alphastats/dataset/dataset.py

```diff
@@ -161,6 +150,37 @@ def _check_loader(loader):
                 "Invalid index_column: consider reloading your data with: AlphaPeptLoader, MaxQuantLoader, DIANNLoader, FragPipeLoader, SpectronautLoader"
             )
 
+    def _create_id_dicts(self, sep: str = ";") -> Tuple[dict, dict, dict]:
+        """Create mapprings from gene, protein to feature and from feature to repr."""
+
+        features = self.mat.columns.to_list()
+        gene_to_features_map = defaultdict(lambda: [])
+        protein_to_features_map = defaultdict(lambda: [])
+        feature_to_repr_map = defaultdict(lambda x: x)
```

Comment:
(Claude) The lambda function will never execute since defaultdict's lambda should take no arguments.